app/serializers/pbs/mailing_list_subscriber_serializer.rb

```diff
@@ -10,7 +10,8 @@ module Pbs::MailingListSubscriberSerializer
 
   included do
     schema do
-      property :correspondence_language, item.correspondence_language
+      # TBD: keep this fields name?
```

Comment:
@carlobeltrame sollten wir hier die Kompatibilität bewahren?

Reply:
Ja, fände ich gut. Dann hats halt in Zukunft zwei Felder `language` und `correspondence_language` mit demselben Inhalt. Aber besser als dass bestehende Anbindungen kaputt gehen.